tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -116,7 +116,7 @@ class TransformListener
   rclcpp::Subscription<tf2_msgs::msg::TFMessage>::SharedPtr message_subscription_tf_;
   rclcpp::Subscription<tf2_msgs::msg::TFMessage>::SharedPtr message_subscription_tf_static_;
   tf2::BufferCore & buffer_;
-  std::atomic<bool> stop_thread_{false};
+  std::unique_ptr<rclcpp::executors::SingleThreadedExecutor> executor_;
```

Comment:
Looking at #133, I actually think that we can get rid of both the `stop_thread_` and `executor_` member variables, and just use a local `executor` in `initThread`.

Reply:
If we want to use the local executor, should we go with the approach in #133 to be pushed to ros2, just removing the `stop_thread`?

Reply:
More-or-less.  For *this* PR targeting master, we don't care about ABI breaks, so we can remove both the `stop_thread_` and `executor_` member variables, and just use a local executor.  For #133 targeting dashing, we'll still use the local executor, but we'll leave the `stop_thread_` and `executor_` member variables.  They won't be used, but they'll maintain ABI.  Make sense?